zeppelin-server/src/main/java/org/apache/zeppelin/service/NotebookService.java

```diff
@@ -313,6 +313,26 @@ public Note importNote(String notePath,
     }
   }
 
+  public boolean runParagraph(String noteId,
```

Comment:
I, basically, believe that it could make others confused. Could you please revert this code to original one if it doesn't have a concrete benefit?

Reply:
Ok, I revert the code.